malloc_count.c

```diff
@@ -122,6 +125,30 @@ extern void malloc_count_reset_peak(void)
     peak = curr;
 }
 
+/* user function to return total allocation volume */
+extern size_t malloc_count_total(void)
+{
+    return local_total;
+}
+
+/* user function to reset total allocation volume to 0 */
+extern void malloc_count_reset_total(void)
+{
+    local_total = 0;
```

Comment:
Well actually just tracking the number of allocations (without ever resetting them) would be valid